packages/esm-patient-clinical-view-app/src/clinical-view-overview/clinical-view-overview.test.tsx

```diff
@@ -4,6 +4,7 @@ import userEvent from '@testing-library/user-event';
 import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
 import { mockPatient } from '../../../../__mocks__/patient.mock';
 import ClinicalViewOverview from './clinical-view-overview.component';
+import { ExtensionSlot } from '@openmrs/esm-framework';
```

Comment:
If what you're asserting is the appearance of the `Edit View` button and the disappearance of the `Add View` button when the user clicks on a clinical view, you do not need this import.